src/tracks/SampleTrack.cpp

```diff
@@ -587,7 +587,12 @@ SampleTrack::SampleTrack( TrackContainer* tc ) :
 							tr( "Volume" ) ),
 	m_panningModel( DefaultPanning, PanningLeft, PanningRight, 0.1f,
 					this, tr( "Panning" ) ),
-	m_audioPort( tr( "Sample track" ), true, &m_volumeModel, &m_panningModel, &m_mutedModel )
+	m_audioPort( tr( "Sample track" ), true, &m_volumeModel, &m_panningModel, &m_mutedModel ),
+	m_recordingChannelModel(RecordingChannel::None,
```

Comment:
The order of declaration and initialization differs. It causes compilation warning. This is why Travis failed.

Reply:
I've got overwhelmed with the amount of warnings clang had threw on me (For some reason, `-Weverything` was on)